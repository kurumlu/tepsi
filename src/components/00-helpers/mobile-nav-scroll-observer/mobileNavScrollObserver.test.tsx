import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import {
  MobileNavScrollObserver,
  MobileNavScrollObserverProps,
} from './MobileNavScrollObserver';

describe('<MobileNavScrollObserver />', () => {
  it('renders with default props', () => {
    const component: ShallowWrapper<MobileNavScrollObserverProps> = shallow(
      <MobileNavScrollObserver forceNavToBeShown />
    );

    expect(component).toMatchSnapshot('default');
  });
});
