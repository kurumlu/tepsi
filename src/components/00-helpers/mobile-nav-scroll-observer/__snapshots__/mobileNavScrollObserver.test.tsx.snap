// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MobileNavScrollObserver /> renders with default props: default 1`] = `
<span
  style={
    Object {
      "display": "none",
    }
  }
/>
`;
